levelkdev/swapr-dapp
# Feat/lp zapping

# Summary

Fixes #491 

Beta version of LP-Zapping feature!

***Important node***
I'm aware of plenty of small bugs, mainly UX, but I believe these are not crucial and can (and should!) be solved one by one.

Since this is quite complex feature there still can be bugs which I didn't see, so keep in mind this is still Beta version!
Designs still needs to be created and as soon as we have them will be added.
Zap contract also needs to be reviewed and the newest version can be found here: https://github.com/ImpeccableHQ/swapr-lp-zapping/pull/2


  # To Test

1. Open Swapr
2. Go to Zap page (go through Liquidity page as well)
3. Select token / pair for zapping
4. Check approval flow
5. Check if transactions passed/failed
6. Check button/popup texts and colors 
7. Play around and have fun
